recipe/bld.bat

```diff
@@ -5,10 +5,12 @@ if "%ARCH%"=="32" (
 )
 
 REM Configure step
-perl configure %OSSL_CONFIGURE% ^
-    --prefix=%LIBRARY_PREFIX% ^
+perl configure %OSSL_CONFIGURE%   ^
+    --prefix=%LIBRARY_PREFIX%     ^
     --openssldir=%LIBRARY_PREFIX% ^
-    shared ^
+    no-module                     ^
+    enable-legacy                 ^
```

Comment:
LGTM but I'd love if @isuruf could chime in as well. He has an eye for possible failures :-)

Reply:
Gentle ping @isuruf. Can explain the context around openssl 3 providers if desired.

Reply:
He is probably too busy with grad school. Let's merge and observe, if something bad happens we pull the package.